packages/core/redux/serialize.js

```diff
@@ -7,27 +7,27 @@ import Immutable from 'immutable'
 import { createTransform } from 'redux-persist'
 import jsan from 'jsan'
 import BigNumber from 'bignumber.js'
-import Amount from '@chronobank/core/models/Amount'
-import BalanceModel from '@chronobank/core/models/tokens/BalanceModel'
-import BalancesCollection from '@chronobank/core/models/tokens/BalancesCollection'
-import TxModel from '@chronobank/core/models/TxModel'
-import TransactionsCollection from '@chronobank/core/models/wallet/TransactionsCollection'
-import OwnerModel from '@chronobank/core/models/wallet/OwnerModel'
-import OwnerCollection from '@chronobank/core/models/wallet/OwnerCollection'
-import MultisigWalletPendingTxModel from '@chronobank/core/models/wallet/MultisigWalletPendingTxModel'
-import TxExecModel from '@chronobank/core/models/TxExecModel'
-import AddressModel from '@chronobank/core/models/wallet/AddressModel'
-import AddressesCollection from '@chronobank/core/models/wallet/AddressesCollection'
-import MultisigWalletCollection from '@chronobank/core/models/wallet/MultisigWalletCollection'
-import DerivedWalletModel from '@chronobank/core/models/wallet/DerivedWalletModel'
-import AllowanceCollection from '@chronobank/core/models/wallet/AllowanceCollection'
-import AllowanceModel from '@chronobank/core/models/wallet/AllowanceModel'
-import MainWalletModel from '@chronobank/core/models/wallet/MainWalletModel'
-import AccountModel from '@chronobank/core/models/wallet/persistAccount/AccountModel'
-import WalletModel from '@chronobank/core/models/wallet/WalletModel'
-import TxHistoryModel from '@chronobank/core/models/wallet/TxHistoryModel'
-import MultisigEthWalletModel from '@chronobank/core/models/wallet/MultisigEthWalletModel'
-import { SignerMemoryModel } from '@chronobank/core/models'
+import Amount from '../models/Amount'
```

Comment:
What about `import * as Models from '../models'`?
There is `index.js` file with all required models.

And below at lines 62-84 just add 
```
if (value instanceof Models.AccountModel) return null
...
```
instead of 
```
if (value instanceof AccountModel) return null
...
```